src/prometheus_alert_rules/disk.rules

```diff
@@ -19,7 +19,7 @@ groups:
           LABELS = {{ $labels }}
         The 6-hour-ahead prediction is made as a linear regression from the last 60 minutes of data.
   - alert: HostDiskSpace
-    expr: used_disk_space{mountpoint=~"/"} > 90
+    expr: used_disk_space{fstype!~"fuse.lxcfs|fuse.snapfuse", mountpoint!~"/run.*|/proc|/dev"} > 90
```

Comment:
- We should probably escape the `.` in `fuse.lxcfs` and `fuse.snapfuse`?
- Add comment explaining why they're excluded?

Reply:
Done